pkg/constants/constants.go

```diff
@@ -46,6 +46,8 @@ const (
 	NutanixCredentialsName = "nutanix-credentials"
 	EksaLicenseName        = "eksa-license"
 	EksaPackagesName       = "eksa-packages"
+	// UpgraderConfigMapName is the name of config map that stores the upgrader images.
+	UpgraderConfigMapName = "upgrader-config-map"
```

Comment:
`config-map` in the name is a bit redundant since it's of type configmap. How about this instead?
```suggestion
	UpgraderConfigMapName = "in-place-upgrade"
```